### ISSUE TITLE
improve user feedback

### ISSUE THREAD

**Issue by LKINSEY** [LKINSEY/MetaDataGUI#11]
Better ways of displaying what's been processed already on Y and what's not
i.e. x for not processed yet and check mark for processed in date drop down for each mouse

**Comment by LKINSEY** [comment]
would be cool to have similar feedback for what got sent to aws already and what didn't as well

**Comment by rozmar** [comment]
I agree, that's an important feature. Maybe that's the best way for feedback and not the command-line output like stuff.
It would be awesome, if it could also identify if something is in progress.
What I suggest is to generate a temporary file in the folder that has the status of each step. When you start copying, the copying script updates the file that it started copying at this give time, and when finishes, it updates the file that the copying ended at this and that time. 
If it errors out, you update the error message.
```
progress_json = {'move_files':{'started': True/False, 
                                             'start_time':datetime,
                                             'end_time':datetime,
                                             'error_message':string},
                           'export_behavior'{'started': True/False, 
                                             'start_time':datetime,
                                             'end_time':datetime,
                                             'error_message':string}
                            ....}
```
And the GUI could read this json file from time to time to get updates
moving files -    finished -   at 5:10pm, error: ''
extract behavior   -    running     -   since 5:10pm
staging videos  ... etc etc

**Comment by LKINSEY** [comment]
The new merge when it happens will be great :) The new updates will focus on user feedback

**Comment by LKINSEY** [comment]
sneak peak into some fun updates 
![image](https://github.com/user-attachments/assets/1176b31b-6933-41cd-a978-601b203cf827)
